tests/acceptance/AdminTablesFilter.feature

```diff
@@ -22,7 +22,7 @@ Feature: filter tables on admin page
   Scenario: Dataset authors table
     Given I am on "/adminDatasetAuthor/admin"
     When I fill in the field of "name" "DatasetAuthor[doi_search]" with "100006"
-    And I press return on the element "(//input)[1]"
+    And I press return on the element "(//input)[2]"
```

Comment:
this changed is needed because the new layout contains an input so the targeted input is no longer the first one in the page